specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/virtualWan.json

```diff
@@ -1675,6 +1675,240 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/routeMaps/{routeMapName}": {
+      "get": {
+        "operationId": "RouteMaps_Get",
+        "x-ms-examples": {
+          "RouteMapGet": {
+            "$ref": "./examples/RouteMapGet.json"
+          }
+        },
+        "description": "Retrieves the details of a RouteMap.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the RouteMap's resource group."
+          },
+          {
+            "name": "virtualHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VirtualHub containing the RouteMap."
+          },
+          {
+            "name": "routeMapName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the RouteMap."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of the VirtualHub retrieved.",
+            "schema": {
+              "$ref": "#/definitions/VirtualHub"
```

Comment:
Do we need a definition of routemap

Reply:
Please fix .. GET and PUT need to return the same item